spec/spec.md

```diff
@@ -696,15 +703,16 @@ One such registry is [provided by this specification](registry/index.html#gatewa
 
 ### Retained DID Set
 
-A [[ref:Retained DID Set]] refers to the set of DIDs a [[ref:Gateway]] retains and republishes to the DHT. A
+A [[ref:Retained DID Set]] refers to the set of DIDs a [[ref:Gateway]] retains and [[ref:republishes]] to the DHT. A
 [[ref:Gateway]] may choose to surface additional [APIs](#gateway-api) based on this set, such as providing a
 [type index](#type-indexing).
 
 To safeguard equitable access to the resources of [[ref:Gateways]], which are publicly accessible and potentially
 subject to [a high volume of requests](#rate-limiting), we suggest an ****OPTIONAL**** mechanism aimed at upholding
-fairness in the retention and republishing of record sets by [[ref:Gateways]]. This mechanism, referred to as a
-[[ref:Retention Proof]], requires clients to generate a proof value for write requests. This process guarantees that
-the amount of work done by a client is proportional to the duration of data retention and republishing a [[ref:Gateway]]
+fairness in the retention and [[ref:republishing]] of record sets by [[ref:Gateways]]. This mechanism, referred to as a
```

Comment:
note much of this language will change in a follow-up PR for #74

Reply:
you may want to include the reasoning being to defend against spam, and specifically write (or create/update) spam, idk just an idea

Reply:
the section has a link to the section (#rate-limiting) which does this